### PR TITLE
DWR-542, validation in organization processor

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/ingest/common/util/ParserHelper.java [hunk 1]
```diff
@@ -60,7 +60,9 @@ public String validate(final String name, final String rawValue, final int maxLe
 
         final String validatedString = rawValue.trim();
 
-        if (validatedString.length() > maxLength) elementErrorCollector.add(new DataElementError(name, rawValue, "string is too long, max length is " + maxLength));
+        if (validatedString.length() > maxLength) {
```
Comment (malaffoon) [inline common/src/main/java/org/opentestsystem/rdw/ingest/common/util/ParserHelper.java]: unrelated formatting change

File: package-processor/src/test/resources/organization.json [hunk 2]
```diff
@@ -125,8 +125,8 @@
       {
         "id": "58ec64cbe4b07da58f50825d",
         "entityId": "DS1111",
-        "entityName": "NoParent",
-        "parentEntityId": "DoesNotExist",
+        "entityName": "Charter",
```
Comment (malaffoon) [inline package-processor/src/test/resources/organization.json]: changed so that one of the samples actually processes cleanly

File: package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultOrganizationProcessor.java [hunk 3]
```diff
@@ -134,8 +142,8 @@ private void processNode(final JsonNode node) {
             if (node.isArray() || (node.isContainerNode() && !node.has("entityType"))) {
                 node.forEach(this::processNode);
             } else if (node.isContainerNode() && node.has("entityType")) {
-                final String entityId = node.get("entityId").asText();
-                final String entityName = node.get("entityName").asText();
+                final String entityId = parserHelper.validate("entityId", node.get("entityId").asText(), 40, false);
+                final String entityName = parserHelper.validate("entityName", node.get("entityName").asText(), 100, false);
                 if (!entityId.isEmpty() && !entityName.isEmpty()) {
```
Comment (agorina) [inline package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultOrganizationProcessor.java]: Could either value be NULL here?

Comment (malaffoon) [inline package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultOrganizationProcessor.java]: Good point, i'll change this and add a test.
